src/Util/ClassSourceManipulator.php

```diff
@@ -404,7 +404,7 @@ private function quoteAnnotationValue($value)
             return 'null';
         }
 
-        if (\is_int($value)) {
+        if (\is_int($value) || '0' === $value) {
```

Comment:
What if we use a scale of "1", wouldn't that still print as `"1"`?

Reply:
You mean type `"1"` in the command console ?

In question helper, all numbers typed are ok and concedered as number, except 0 which is '0' (so not a number)